src/main/java/jade/parser/Parser.java

```diff
@@ -0,0 +1,88 @@
+package jade.parser;
+
+import jade.data.Deadline;
+import jade.data.Event;
+import jade.data.Todo;
+import jade.commands.*;
```

Comment:
Instead of using *, it would be better if imported classes are listed explicitly.